tests/schema/test.firefox-schemas-import.js

```diff
@@ -1171,17 +1162,14 @@ describe('firefox schema import', () => {
         .withArgs('https://hg.mozilla.org/mozilla-central/archive/FIREFOX_AURORA_54_BASE.tar.gz')
         .returns(mockStream);
       expect(fs.readdirSync(outputPath)).toEqual([]);
-      return fetchSchemas({ version: 54, outputPath })
-        .then(() => {
-          expect(true).toBeFalsy();
-        }, () => {
-          // Manually remove the tar file since it doesn't get cleaned up.
-          fs.unlinkSync('tmp/FIREFOX_AURORA_54_BASE.tar.gz');
-          expect(true).toBeTruthy();
-        });
+      await expect(
+        fetchSchemas({ version: 54, outputPath })
+      ).rejects.toThrow();
+      // Manually remove the tar file since it doesn't get cleaned up.
+      fs.unlinkSync('tmp/FIREFOX_AURORA_54_BASE.tar.gz');
```

Comment:
I'm wondering if it would be better to move the unlinking of the temporary files created in this test file in an "teardown" step (https://facebook.github.io/jest/docs/en/setup-teardown.html).

@EnTeQuAk how that sounds to you?